apis/eks/v1beta1/zz_nodegroup_types.go

```diff
@@ -27,12 +27,6 @@ type AutoscalingGroupsParameters struct {
 
 type LaunchTemplateInitParameters struct {
 
-	// Identifier of the EC2 Launch Template. Conflicts with name.
-	ID *string `json:"id,omitempty" tf:"id,omitempty"`
-
-	// Name of the EC2 Launch Template. Conflicts with id.
-	Name *string `json:"name,omitempty" tf:"name,omitempty"`
-
```

Comment:
@lsviben, Is it expected to be removed from InitParameters when we add a reference to an argument?

Reply:
yes, references are not a part of initProvider, ref: https://github.com/upbound/upjet/blob/14cd59c9986d2a6bc07ea8c0282cde4cad504644/pkg/types/field.go#L282

Reply:
Thanks for the information.